src/client-common/client-telemetry-helper.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Maybe this module can be renamed to telemetry-helper.js instead of the current name.
No matter if this module is going to be under the current folder, or under src/modules, I think it is clear that this is for the client.